src/screens/WhotoFollow/elements/ItemUser.js

```diff
@@ -1,13 +1,14 @@
 import * as React from 'react';
 import {Animated, Pressable, StyleSheet, View} from 'react-native';
 
+import PropTypes from 'prop-types';
 import IconAdd from '../../../assets/icon/IconAdd';
 import IconCheck from '../../../assets/icon/IconCheck';
 import UserInfo from './UserInfo';
 import dimen from '../../../utils/dimen';
 import {colors} from '../../../utils/colors';
 
-const ItemUser = ({photo, username, bio, followed, onPress, userid}) => {
+const ItemUser = ({photo, username, bio, followed, onPress, userid, karmaScore}) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `karmaScore` has been added to the `ItemUser` component. Ensure that this prop is being passed correctly from the parent component and that it has a default value or is marked as required in PropTypes to avoid potential issues with undefined values.
```diff
- const ItemUser = ({photo, username, bio, followed, onPress, userid, karmaScore}) => {
+ const ItemUser = ({photo, username, bio, followed, onPress, userid, karmaScore = 0}) => {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
ignore